db/change_cache_test.go

```diff
@@ -1986,7 +1986,7 @@ func BenchmarkProcessEntry(b *testing.B) {
 			defer context.Close()
 
 			changeCache := &changeCache{}
-			if err := changeCache.Init(context, nil, nil); err != nil {
+			if err := changeCache.Init(context, nil, nil, nil); err != nil {
```

Comment:
Passing in a `nil` channel here means the goroutine in Init will block forever trying to read from it, which doesn't seem correct.

Reply:
Check for `nil` channel here, for tests.